### PR TITLE
Add markdown field documentation

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
Add markdown field documentation

#### Ticket Link
Depends on:
https://github.com/mattermost/mattermost-mobile/pull/5407
https://github.com/mattermost/mattermost-webapp/pull/8171
https://github.com/mattermost/mattermost-plugin-apps/pull/192

### CODE REVIEW

**Review by hanzei — APPROVED** [review]

File: site/content/integrate/apps/interactivity/_index.md [hunk 1]
```diff
@@ -24,14 +24,17 @@ They are defined by:
...
-| `bool`           | A boolean selector represented as a checkbox.          |
-| `user`           | A dropdown with to select users.                       |
-| `channel`        | A dropdown to select channels.                         |
+| Name             | Description                                              |
+| :--------------- | :------------------------------------------------------- |
+| `text`           | Text field.                                              |
+| `static_select`  | A dropdown select with static elements.                  |
+| `dynamic_select` | A dropdown select that loads the elements dynamically.   |
+| `bool`           | A boolean selector represented as a checkbox.            |
+| `user`           | A dropdown with to select users.                         |
```
Comment (justinegeffen) [inline site/content/integrate/apps/interactivity/_index.md:33]:
```suggestion
| `user`           | A dropdown to select users.                         |
```

**Review by justinegeffen — APPROVED** [review]

File: site/content/integrate/apps/interactivity/_index.md [hunk 2]
```diff
@@ -24,14 +24,17 @@ They are defined by:
...
+| :--------------- | :------------------------------------------------------- |
+| `text`           | Text field.                                              |
+| `static_select`  | A dropdown select with static elements.                  |
+| `dynamic_select` | A dropdown select that loads the elements dynamically.   |
+| `bool`           | A boolean selector represented as a checkbox.            |
+| `user`           | A dropdown to select users.                              |
+| `channel`        | A dropdown to select channels.                           |
+| `markdown`       | Only visible on modal forms. An arbitrary markdown text. |
+
+Markdown fields are a special field that allows you to better format your form. They will not generate any value towards the app. The content is defined in the field description.
```
Comment (mickmister) [inline site/content/integrate/apps/interactivity/_index.md:37]:
```suggestion
Markdown fields are a special field that allows you to better format your form. They will not generate any value in the form submission sent to the app. The content is defined in the field description.
```

Comment (mickmister) [reply site/content/integrate/apps/interactivity/_index.md:37]: Why is the content defined in the description? Seems reasonable

**Review by mickmister — APPROVED** [review]

File: site/content/integrate/apps/interactivity/_index.md (hunk 2 above)
Comment (larkox) [reply site/content/integrate/apps/interactivity/_index.md:37]: The only reason I considered was lack of a better place :sweat_smile:

**Pull request merged**